xml/System/AggregateException.xml

```diff
@@ -682,8 +682,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Returns the <see cref="T:System.AggregateException" /> that is the root cause of this exception.</summary>
-        <returns>The <see cref="T:System.AggregateException" /> that is the root cause of this exception.</returns>
+        <summary>Returns the <see cref="Exception"/> that is the root cause of this exception. This will either be the root exception, or the first <see cref="AggregateException"/> that contains either multiple inner exceptions or no inner exceptions at all.</summary>
```

Comment:
Same comment as in https://github.com/dotnet/runtime/pull/107743#discussion_r1757134862 : It is not clear what "root exception" means here.

Reply:
@jkotas how would you make this clearer?

Reply:
[Made a suggestion](https://github.com/dotnet/dotnet-api-docs/pull/10396#discussion_r1765540926)